src/XProfile/VirtoCommerce.XProfile/Schemas/UserType.cs

```diff
@@ -0,0 +1,37 @@
+using GraphQL.Types;
+using VirtoCommerce.Platform.Core.Security;
+
+namespace VirtoCommerce.ExperienceApiModule.XProfile.Schemas
+{
+    public class UserType : ObjectGraphType<ApplicationUser>
+    {
+        public UserType()
+        {
+            Field(x => x.AccessFailedCount);
```

Comment:
Add `ContactType` field to the `UserType` with resolving as well as extend `ContactType` with the `SecurityAccounts` fields that have type IList<UserType>